src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -57,7 +57,7 @@
       <DependentUpon>NuGetCommand.resx</DependentUpon>
     </Compile>
     <EmbeddedResource Update="NuGetResources.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
+      <Generator>PublicResXFileCodeGenerator</Generator>
```

Comment:
What version of VS 2017 are you using to build this project, it may be the issue since of me using the Dogfood build of VS 2017. When I build the solution that built the .resx. and without the csproj change it defaulted the class to be internal.